code/index.html

```diff
@@ -23,25 +23,56 @@
       <h1>Does the person have</h1>
 
       <select id="questions">
+          <!-- Options of hair colors  -->
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
```

Comment:
`<option value="brown" selected>brown hair</option>` should fix that this option is not active before you deselect and select it again. 🤓